spec/models/test_track/web_session_spec.rb

```diff
@@ -513,17 +513,22 @@ def current_clown; end
     let(:v1_split_registry) do
```

Comment:
Not sure to what we should rename this.

Reply:
we can just call it split registry now that there is only a single version